package-lock.json

```diff
@@ -5004,7 +5006,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
I think these are unimportant changes that a recent version of npm started to like to prefer.